tools/setup_helpers/extension.py

```diff
@@ -84,10 +90,10 @@ def build_extension(self, ext):
         # However, the following `cmake` command will build all of them at the same time,
         # so, we do not need to perform `cmake` twice.
         # Therefore we call `cmake` only for `torchaudio._torchaudio`.
-        if ext.name != "torchaudio._torchaudio":
+        if ext.name != "torchaudio.lib.libtorchaudio":
             return
 
-        extdir = os.path.abspath(os.path.dirname(self.get_ext_fullpath(ext.name)))
+        extdir = os.path.abspath(os.path.dirname(self.get_ext_fullpath("torchaudio.foo")))
```

Comment:
could be helpful to add a comment here that explains why we use "torchaudio.foo"

Reply:
Could be anything as the last path part is being removed by `dirname`.
However, can we put all the C++ extension in `lib`?
Currently extension modules like `torchaudio._torchaudio` is put directly under the top module.
We will have three of these, so I am thinking that putting all of them in `torchaudio.lib` make it cleaner.

Reply:
sounds reasonable to me